### ISSUE TITLE
Geomap Route: part of route connect to the wrong repeat

### ISSUE THREAD

**Issue by zbynekwinkler** [grafana/grafana#93827]
### What happened?

I have added geomap with satellite position and route display instead of markers and forward arrows:

![Image](https://github.com/user-attachments/assets/70d582e6-0914-4c11-90fd-3562b674cc2e)

The route repeats where the map repeats but at one point the route connects to the wrong repeat.

### What did you expect to happen?

I expect route arrows to connect to the correct repeat.

### Did this work before?

I don't know.

### How do we reproduce it?

1. Add geomap panel
2. Add layer with satellite position
3. Switch to route + forward arrows

Example of the data (infinity plugin csv):

```csv
timestamp,lat,lon,height
1727348907681,3.2663029221362545,161.14013781846856,466.378768832511
1727348967681,7.093320991994195,160.39318975031125,465.7397819621392
1727349027681,10.919942764093886,159.63788107176725,465.2788477679551

```

### Environment (with versions)?

Grafana: 11.2.0
OS: Debian 12
Browser: Edge on Windows 11

### Grafana platform?

Docker

### Datasource(s)?

Infinity plugin with a csv file

**Comment by zwn** [comment]
As the satellite moves it gets even stranger around the south pole:

![Image](https://github.com/user-attachments/assets/b32b69c3-e4b8-4d66-bb43-110a40494e72)

**Comment by realAudi** [comment]
@zbynekwinkler 
have you heard anything back on this, has someone looked at this report?

**Comment by jnicholsDLR** [comment]
It appears to be related to the longitude switching from negative to positive, which I detailed in this bug report some time ago https://github.com/grafana/grafana/issues/81196

**Comment by realAudi** [comment]
@jnicholsDLR 
I agree that might be the case in the first instance with the satellite path, but in the case I pointed out I do not cross the pacific, yet it tries to close the path. 
Perhaps these are 2 different cases in fact. Still waiting to hear something back on the ship issue
